client/src/models.ts

```diff
@@ -36,3 +38,31 @@ export interface User {
     readonly username: string;
 
 };
+
+export interface RestaurantProperties {
+    readonly name: string;
+    readonly address: string;
+    readonly is_approved: boolean;
+    readonly average_rating: number | null;
+}
+
+type NoBbox<T> = Omit<T, "bbox" | "id">;
```

Comment:
Given the name of `NoBbox` I'd expect this to _just_ be an alias for `Omit<T, "bbox">`.

Note: this change on it's own will cause a type error bc `featureDecoder` in `decoders.ts` doesn't know about `id` - you will need to update the decoder as well.
```suggestion
type NoBbox<T> = Omit<T, "bbox">;
```